http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -339,7 +339,13 @@ class HttpSamDAO[F[_]: Effect](httpClient: Client[F], config: HttpSamDaoConfig,
           )
         )(s => Effect[F].pure(s))
       )
-      authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, token))
+      resp <- getUserSubjectIdFromToken(token)
+    } yield resp
+
+  override def getUserSubjectIdFromToken(token: String)(implicit ev: Ask[F, TraceId]): F[Option[UserSubjectId]] = {
```

Comment:
Given an invalid token will this return `F[None]` or an error?

Reply:
So I think if the token is invalid, the previous google auth call will fail. This will return `None` (sam call returns 404) if the user is not found

Reply:
Ah, ok